### ISSUE TITLE
how to select sequence name, and delete some columns?

### ISSUE THREAD

**Issue by liamxg** [4ment/seqotron#5]
@4ment

**Comment by liamxg** [comment]
<img width="538" alt="image" src="https://github.com/4ment/seqotron/assets/46078579/54b3a4d0-fca1-4e56-bffe-99d6d5e0e76c">

**Comment by liamxg** [comment]
how to paste some sequences to excel?

**Comment by liamxg** [comment]
@4ment

**Comment by 4ment** [comment]
You can't export to excel, you have to modify the alignment if needed and then export it as a fasta file.
For editing, pease have a look at the manual: [https://www.dropbox.com/s/6gvt0l4gmfwggpe/seqotron-manual.pdf](url)

**Comment by liamxg** [comment]
I mean paste sequence name to keyboard, is that possible?

**Comment by 4ment** [comment]
If you select some sequences in the left panel and use Command-C you can then paste the sub-alignment to a text file as a nexus file.

**Comment by liamxg** [comment]
@4ment, yes, i tried before, that is a good way to transform fasta to nexus. 
But if I just want to copy sequence names, is that possible?
